grails-app/domain/org/pih/warehouse/core/Location.groovy

```diff
@@ -285,6 +285,25 @@ class Location implements Comparable<Location>, java.io.Serializable {
         Location.list().findAll { it.isDepotWardOrPharmacy() }.sort { it.name }
     }
 
+    Location getDefaultLocation() {
+        return null
+    }
```

Comment:
Is there something missing? I can't see any comment here

Reply:
Yeah, this is awkward at best. I think this was just a quick hack to get it working and I was expecting to get back to it. I am trying to provide a representation of a default location. But the default location is just a Location that is null. 

I would love to do this in a better way and open to ideas. A javadoc comment is necessary. And maybe this needs to be a static method, returning a constant? 

```
static Location getDefaultLocation() { 
    return Constants.DEFAULT_LOCATION 
}
```
or something like this? 
```
final static Location DEFAULT_LOCATION = null;
static Location getDefaultLocation() { 
    return Location.DEFAULT_LOCATION; 
}
```

Reply:
The first static method returning a constant looks great